### DISCUSSION TITLE
Open ID Connect – How to implement it?

### DISCUSSION THREAD

**Discussion by alexanderkroneis** [saloonphp/saloon#382] [Ideas]
Hi,

we started to migrate to Saloon to get a consistent experience through our whole application. The first integration we started to migrate is required to use the OpenID Connect (OIDC) protocol.

We're wondering if there is a "saloon-native" way to implement that since we're required to send a client id, username and password and the default OAuthConfig does not support that. It's password grant type authentication.

Looking forward to your response 🎉

**Comment by paulheijman** [comment]
Hi Alexander,

I'm having the same issue. Did you ever managed to get it working?

Regards,

Paul
